profile.d/security

```diff
@@ -25,4 +25,10 @@ RELEASE_GIT_PRODUCTION_BRANCH=master
 # Remark: Version to be promoted will be the latest version pushed to the maven repository
 #         as retrieved from ./utils/getJenkinsVersion.py
 
+# Unused:
 MAVEN_REPOSITORY_PRODUCTION_NAME=releases
+
+# Real parameters:
+MAVEN_REPOSITORY_NAME=cassie
+RELEASE_GIT_BRANCH=security-stable-2.277
+JENKINS_VERSION=2.277.2
```

Comment:
```suggestion
JENKINS_VERSION=2.277.2
RELEASELINE=-stable
```

Reply:
Release line is needed by the packaging script to now where to publish the release either get.jenkins.io/debian or get.jenkins.io/debian-stable